deployment/templates/fluentd.yaml

```diff
@@ -84,6 +84,9 @@ objects:
             - name: dockerhostname
               mountPath: /etc/docker-hostname
               readOnly: true
+            - name: throttle-settings
```

Comment:
they shouldn't need to... they'll be pulling in the latest image, so that should cover the logic behind it and then it would be adding the volume mount to the DC (working on docs right now on how to do that).  Unless I'm missing something, it shouldn't be much more complicated than that